src/browser/modules/DBMSInfo/styled.tsx

```diff
@@ -34,6 +34,7 @@ const chip = styled.div`
   span {
     line-height: normal;
   }
+  display: inline-block;
```

Comment:
This change allows us to reuse the sidebar styling of nodes/relationships in the new panel